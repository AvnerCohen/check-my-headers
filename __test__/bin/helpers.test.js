const { printError, printInfo, printSuccess } = require('../../bin/helpers')

const consoleOrg = console.log
const consoleError = console.error
beforeEach(() => {
  console.log = () => {}
  console.error = () => {}
})

afterEach(() => {
  console.log = consoleOrg
  console.error = consoleError
})

describe('General CLI helpers', () => {
  test('Terminal format helpers (colors)', () => {
    expect(printError('This is an error msg')).toMatchSnapshot()
    expect(printInfo('This is an information msg')).toMatchSnapshot()
    expect(printSuccess('This is a success msg')).toMatchSnapshot()
  })
})
