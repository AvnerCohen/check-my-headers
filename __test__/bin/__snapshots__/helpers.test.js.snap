// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`General CLI helpers Terminal format helpers (colors) 1`] = `undefined`;

exports[`General CLI helpers Terminal format helpers (colors) 2`] = `undefined`;

exports[`General CLI helpers Terminal format helpers (colors) 3`] = `undefined`;
